templates/messages_notification.html

```diff
@@ -350,7 +350,156 @@
         padding: 0px 0px 0px 12px !important;
       }
     }
+
+    .slackAttachmentContent {
```

Comment:
Would you mind renaming these `messageAttachmentX`? We've moved everything outside the server code to refer to these as "message attachments", and while renaming it everywhere is blocked by being a breaking change for people using the Go API, I think we probably want to use the new name in new code

Reply:
Done!

Reply:
Thanks!